scripts/plugins/firewall-port

```diff
@@ -0,0 +1,38 @@
+#!/bin/bash
+#
+# Copyright (c) Citrix Systems 2017. All rights reserved.
+
+set -e
+
+# Use this script to open/close port. 
+
+
+OP="$1"
+PORT="$2"
+CHAIN="xapi-INPUT"
+RULE="-p tcp -m conntrack --ctstate NEW -m tcp --dport $PORT -j ACCEPT"
+
+case "${OP}" in
+    open)
+        if ! iptables -C $CHAIN $RULE > /dev/null 
+        then
+            if ! iptables -S $CHAIN > /dev/null 
+            then
+                iptables -N $CHAIN
+                iptables -I INPUT -j $CHAIN
+            fi
+        iptables -I $CHAIN $RULE
+        service iptables save
+        fi
+        ;;
+    close)
+        iptables -D $CHAIN $RULE
```

Comment:
Ah, right . when close 6640 twice, it will return non-zero exit code. update to check when close.